pages/index.js

```diff
@@ -16,29 +19,48 @@ export default function Home() {
 
       <Main>
         <h1>My App</h1>
-        <SearchbarLostItem />
+        <div>
+          <h4>Search1</h4>
+          <input
+            placeholder="What are you looking for?"
+            onChange={(event) => setSearch1(event.target.value)}
+          ></input>
```

Comment:
Your `input` needs a corresponding `label` for accessibility reasons. More [info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)
For the label to then work properly, the input needs an `id` and the label an `htmlFor`.
The input is self-closing -> you don't need an opening and closing tag.